### DISCUSSION TITLE
Album/ Folder Location, geotag question

### DISCUSSION THREAD

**Discussion by Rathna-K** [photoprism/photoprism#3129] [Q&A]
Hi there, I am trying out photoprism and have a question as I am unable to locate any info in the documentation. 
1. I see a "Location" textbox when I edit a folder/ album. is this is geotag location? Meaning if I enter an address or lot/lng, will it apply to all the photos and videos to that album? 
2. if not, is there another way where I can apply geotag to a batch of photos and videos without using Lightroom or some other apps? 
thank you,
RK

**Comment by graciousgrey** [comment]
>  I see a "Location" textbox when I edit a folder/ album. is this is geotag location? Meaning if I enter an address or lot/lng, will it apply to all the photos and videos to that album?
 
This is only a text field that can be used to describe the album.

> If not, is there another way where I can apply geotag to a batch of photos and videos without using Lightroom or some other apps?

Not yet. Batch edit as well as writing metadata back to files is planned. For now you could use Exiftool for this and re-index your library.

**Reply by Rathna-K** [reply]
Would be nice if it recognized the coordinates or even enable address look up feature and add to all photos in the album. Same multi-select and apply for some photos as well!
